### PR TITLE
Quo2: keypair blur

### LINKED ISSUE #17068: Follow up: Keypair
PR: https://github.com/status-im/status-mobile/pull/17019

- [ ] Fix dot
- [ ] Add blur

### PR DESCRIPTION
**Pull request by OmarBasem**
fixes: https://github.com/status-im/status-mobile/issues/17068

This PR implements blur for Keypair component.

[Designs](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=24100-238159&mode=design&t=fQ9Sv9dm424lLEW6-4)

### CODE REVIEW

File: src/quo2/components/wallet/keypair/view.cljs
Comment (smohamedjavid) [inline src/quo2/components/wallet/keypair/view.cljs]: `assoc` is the best choice here.

File: src/quo2/components/wallet/keypair/view.cljs
Comment (smohamedjavid) [inline src/quo2/components/wallet/keypair/view.cljs:95]: @OmarBasem - Since we pass the whole `props` to `details-view`. We need to de-structure the map in the `details-view` component.

File: src/quo2/components/wallet/keypair/view.cljs
Comment (OmarBasem) [reply src/quo2/components/wallet/keypair/view.cljs]: @smohamedjavid fixed it

**Review by smohamedjavid — APPROVED** [review]
🚀

**Comment by OmarBasem** [comment]
@Francesca-G a design review please :)

**Review by Francesca-G — APPROVED** [review]
Looks good :)

**Comment by J-Son89** [comment]
> @Francesca-G a design review please :)

Just a suggestion, we should hold off getting design reviews/QA for some reasonable time period, even if we have approvals.  This pr is only open a few hours and some of our devs have not even started their day yet so it makes it harder to consider their requested changes once the review process has began. 1 dev's approval is the minimum requirement but we agreed upon the meetings with @Samyoul that we would leave a reasonable timeframe for others to review. (f I recall correctly)

File: src/quo2/components/wallet/keypair/view.cljs
Comment (J-Son89) [inline src/quo2/components/wallet/keypair/view.cljs]: just a note, we made a slight adjustment to the docs so that this will be `:size-32` after this pr is merged: 
https://github.com/status-im/status-mobile/pull/17279

**Review by J-Son89 — APPROVED** [review]

**Review by vkjr — APPROVED** [review]

**Comment by jakubgs** [comment]
You really shouldn't merge PRs with merge conflicts that break iOS builds dude:
https://github.com/status-im/status-mobile/blob/1aadffd3009e757ccb36600469cadad1f5e5b6db/ios/Podfile.lock#L775-L783

**Comment by OmarBasem** [comment]
@jakubgs my bad, apologies, made a PR https://github.com/status-im/status-mobile/pull/17406

**Pull request merged**
